practice-app/website/test/equipment_test.py

```diff
@@ -0,0 +1,27 @@
+import unittest
+import requests, json
+
+class TestEquipmentAPI(unittest.TestCase):

```

Comment:
As discussed on the team meeting, unit test should not make requests to endpoints. Please update unit tests.

Reply:
Done, thanks!